hub/apps/design/globalizing/design-for-bidi-text.md

```diff
@@ -30,7 +30,7 @@ Four elements influence BiDi design decisions in Windows.
 
 ## Feature design overview
 
-Windows supports the four BiDi design elements. Let's look at some of the major relevant features in Windows, and provide some context around how they affect your app.
+Windows supports the four BiDi design elements. Let's look at some of the major relevant features of (prior versions of) Windows, and provide some context around how they affect your app.
```

Comment:
Prior to what? Is this not current?

Reply:
The screenshots describe sometimes the tiles in Windows 8